kuma/users/views.py

```diff
@@ -680,6 +681,17 @@ def get_form(self, form_class=None):
                     "verified": False,
                     "primary": False,
                 }
+
+        # This magic sauce only exists whilst this Django view is being used
+        # both by Yari (in testing) and by old Kuma (production).
+        # Once Kuma id divorced from doing any
```

Comment:
Delete this comment line?

Reply:
No, we're going to have to come back to this view function soon. I do worry about remembering to do so due to being human and that sickness of not wanting to sink too much time in kuma things for something as unimportant as sign-up.